docs/docsite/rst/conf.py

```diff
@@ -386,3 +392,22 @@
 ]
 linkcheck_workers = 25
 # linkcheck_anchors = False
+
+# Generate redirects for pages when building on Read The Docs
+def setup(app: Sphinx) -> dict[str, str]:
+
+    if tags.has('redirects'):
+
+        redirects = {}
```

Comment:
No need to initialize an empty value only to unconditionally replace it a few lines below.
```suggestion
```